code/modules/projectiles/guns/projectile/explorer_weapons.dm

```diff
@@ -0,0 +1,712 @@
+// Energy weapons for explorers.
+// For those who like pretty lights and the smell of ozone.
+
+
+//The las-rifle is the standard for energy weapons, unlike the standard laser rifle it does not have any armor penetration, \
+nor alternate firing modes. Use this as the base line for laser weapon balance. -Kazkin
+/obj/item/weapon/gun/energy/exlaserrifle
+	name = "\improper expedition las-rifle"
+	desc = "A Void Runner, LLC. produced laser rifle modeled after an existing and old ballistic rifle. What it lacks in function \
+	and armor penetration that a standard laser rifle has it makes up for in durability and price. This one appears to be \
+	missing its safety lock."
+	icon = 'icons/obj/gun_vr.dmi'
+	icon_state = "stg60"
+	wielded_item_state = "bullpup"
+	fire_delay = 8
+	slot_flags = SLOT_BACK
+	w_class = ITEMSIZE_LARGE
+	force = 10
+	origin_tech = list(TECH_COMBAT = 1, TECH_MAGNET = 1)
+	matter = list(DEFAULT_WALL_MATERIAL = 2000)
+	projectile_type = /obj/item/projectile/beam/exlas
+//	one_handed_penalty = 30
+	charge_cost = 240
+
+/obj/item/weapon/gun/energy/exlaserrifle/update_icon()
+	..()
+	if(power_supply)
+		icon_state = "stg60"
+	else
+		icon_state = "stg60-empty"
+
+
+
+
+// An alternative option to the weaker but longer lasting ballistic pistols. The bolter does high damage and has \
+mild armor penetration, greater than the las-rifle, but suffers from massive ammo consumption which makes it \
+strictly a hold out pistol to use in an emergency. -Kazkin
+/obj/item/weapon/gun/energy/exlaserpistol
+	name = "\improper expedition 'bolter'"
+	desc = "Officially named the A38 Hi-Zap pistol this weapon is commonly referred to by its acronym or more \
+	commonly as the 'bolter' pistol. A popular choice for a sidearm given its high intensity shots but commonly \
+	considered inferior due to its rapid ammo consumption. This one appears to be missing its safety lock."
+	icon = 'icons/obj/gun.dmi'
+	icon_state = "oldlaser"
+	item_state = "oldlaser"
+	wielded_item_state = "laser"
+	fire_delay = 6
+	slot_flags = SLOT_BELT|SLOT_HOLSTER
+	w_class = ITEMSIZE_NORMAL
+	force = 8
+	origin_tech = list(TECH_COMBAT = 1, TECH_MAGNET = 1)
+	matter = list(DEFAULT_WALL_MATERIAL = 2000)
+	projectile_type = /obj/item/projectile/beam/bolterlaser
+	charge_cost = 800
+
+
+
+
+// This weapon is probably the most varied and with luck most powerful of the energy option. The burst mode takes a \
+accuracy penalty but if all shots hit an unarmored target it deals the same damage as a laser shotgun. The upside \
```

Comment:
these arent commented out.

Reply:
That issue was its commented out in the dream maker code build because 

// long comment /
comment stop it from dragging far off screen to read easier.

It github it looks incorrect but in dream maker it works fine, I can alter it I suppose, if its that big of an issue.